### PR TITLE
Closes #587 | Add/Update Dataloader MEN

### LINKED ISSUE #587: Create dataset loader for MEN-Dataset
Dataloader name: `men/men.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?men

|     Dataset| men |
|-------------|---|
| Description | The Malaysian English News (MEN) dataset includes 200 Malaysian English news article with human annotated entities and relations (in total 6,061 entities and 3,268 relation instances). Malaysian English combines elements of standard English with Malay, Chinese, and Indian languages. Four human annotators were split into 2 groups, each group annotated 100 news articles and inter-annotator agreement was calculated between 2 or more annotators working on the same task (entity annotation; F1-score 0.82, relation annotation; F1-score 0.51). |
| Subsets | - |
| Languages | eng |
| Tasks | Named Entity Recognition, Relation Extraction |
| License | MIT (mit) |
| Homepage | https://github.com/mohanraj-nlp/MEN-Dataset/tree/main |
| HF URL | - |
| Paper URL | https://arxiv.org/abs/2402.14521 |

### PR DESCRIPTION
**Pull request by patrickamadeus**
Closes #587 

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/{my_dataset}/{my_dataset}.py` (please use only lowercase and underscore for dataset folder naming, as mentioned in dataset issue) and its `__init__.py` within `{my_dataset}` folder.
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_LOCAL`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py` or `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py --subset_id {subset_name_without_source_or_seacrowd_suffix}`.
- [.] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### Test
image

### CODE REVIEW

File: seacrowd/sea_datasets/men/men.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The Malaysian English News (MEN) dataset includes 200 Malaysian English news article with human annotated entities and relations (in total 6,061 entities and 3,268 relation instances). Malaysian English combines elements of standard English with Malay, Chinese, and Indian languages. Four human annotators were split into 2 groups, each group annotated 100 news articles and inter-annotator agreement was calculated between 2 or more annotators working on the same task (entity annotation; F1-score 0.82, relation annotation; F1-score 0.51).
+"""
```
Comment (jen-santoso) [inline seacrowd/sea_datasets/men/men.py:18]: Let's remove these lines, some are comments and some looks like a duplicate from `_DESCRIPTION`

File: seacrowd/sea_datasets/men/men.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                            "text": [articles[article_id][entity["position"]["start_offset"] : entity["position"]["end_offset"]]],
+                            "offsets": [[entity["position"]["start_offset"], entity["position"]["end_offset"]]],
+                            "normalized": [],
+                        }
+                        for entity in entities
+                    ],
+                    "events": [],
+                    "coreferences": [],
+                    "relations": [
+                        {"id": f"{article_id}-relation-{relation['id']}", "type": relation["relation"], "arg1_id": f"{article_id}-entity-{relation['head']}", "arg2_id": f"{article_id}-entity-{relation['tail']}", "normalized": [{"db_name": relation["relation_source"], "db_id": ""}]} for relation in relations
```
Comment (jen-santoso) [inline seacrowd/sea_datasets/men/men.py:195]: Please run the `make check_file` command. This line seems too long.

**Review by jen-santoso — CHANGES_REQUESTED** [review]
Hi @patrickamadeus 
Thank you for implementing / updating the dataloader. 
Dataloader runs well, just a few nits in the comment.

**Comment by patrickamadeus** [comment]
Thank you for the revision @jen-santoso ! Could you kindly review the newest commit?

**Review by jen-santoso — APPROVED** [review]
Oops, sorry for the late reply. thanks for the reminder @holylovenia 
@patrickamadeus : LGTM! Thank you for implementing this dataloader and updating based on the comments! 
merging now

**Pull request merged**